tests/libtest.sh

```diff
@@ -559,6 +559,18 @@ skip_without_user_xattrs () {
     fi
 }
 
+_have_systemd_and_libmount=''
+have_systemd_and_libmount() {
+    if test "${_have_systemd_and_libmount}" = ''; then
+        if [ $(ostree --version | grep -c -e '- systemd' -e '- libmount') -eq 2 ]; then
```

Comment:
Note `ostree --version` is actually valid parsable YAML. I think we make use of that fact in other places already. But heh, I like this.